app/presenters/comment_presenter.rb

```diff
@@ -0,0 +1,46 @@
+class CommentPresenter < ApplicationPresenter
+  presents :comment
```

Comment:
What happens if a comment isn't passed in? You have a lot of logic that is checking if the comment is nil, but I'm not sure if it's necessary.

Reply:
You're right - I was able to eliminate a few safe nav operators just now with no tests breaking and the views render as I expect. In an earlier iteration of the view, it was necessary to account for instances where a parent comments did NOT have children reply comments, but since the block is wrapped with `if presenter.replies`, subsequent cautions aren't needed. Thanks!